### PR TITLE
Remove shell injection in example

### PR DESCRIPTION
**Pull request by soliton-**
### Why:

Fixes code injection in shell code example

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Shell code example

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/actions/creating-actions/dockerfile-support-for-github-actions.md [hunk 1]
```diff
@@ -83,9 +83,10 @@ Using the example Dockerfile above, {% data variables.product.product_name %} wi
 ``` sh
 #!/bin/sh
 
-# `$*` expands the `args` supplied in an `array` individually
-# or splits `args` in a string separated by whitespace.
-sh -c "echo $*"
+# `$#` expands to the number of arguments and `$@` expands to the supplied `args`
+printf '%d args:' "$#"
```
Comment (lucascosti) [inline content/actions/creating-actions/dockerfile-support-for-github-actions.md:87]: Putting quotes around each arg, and removing the blank echo at the end.

```suggestion
printf '%d args:' "$#"
printf $' \'%s\'' "$@"
```

**Review by lucascosti — DISMISSED** [review]
Thanks a lot for this, @soliton-! Just made a slight change and I'll get this merged in

**Review by lucascosti — APPROVED** [review]

File: content/actions/creating-actions/dockerfile-support-for-github-actions.md (hunk 1 above)
Comment (soliton-) [reply content/actions/creating-actions/dockerfile-support-for-github-actions.md:87]: The blank echo is used to print a newline after the arguments. Could also use `printf '\n'`.

It looks like what was merged is a mix of both now. I don't mind using single quotes around the arguments but `$''` is not POSIX. Could use `printf " '%s'" "$@"` instead.

Comment (lucascosti) [reply content/actions/creating-actions/dockerfile-support-for-github-actions.md:87]: Thanks a lot, @soliton-. I also applied my change to the wrong lines 🤦‍♂️. Fixed in this PR: https://github.com/github/docs/pull/22635

**Pull request merged**